### PR TITLE
🐛 Fix bugs in example scenario

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR fixes bugs discovered while testing the nascent placement translator (#245).

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
Testing reminded me that Kubernetes deserialization converts `yes` and `no` to boolean values, which are not acceptable in positions where the schema says a string must appear.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md [hunk 1]
```diff
@@ -233,6 +233,7 @@ metadata:
 spec:
   reference:
     export:
+      path: "root:compute"
```
Comment (ezrasilvera) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md:236]: Is  that related to the si/non replacement ?

Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md:236]: No, it is fixing a distinct bug.

**Pull request merged**